.circleci/config.yml

```diff
@@ -267,7 +278,10 @@ workflows:
       - golang-lint
       - golang-modules-tidy
       - golang-test
-      - golang-validate-genesis
+      - golang-validate-genesis:
+          matrix:
+            parameters:
+              chainid: ["7777777", "4801", "1750", "480","34443", "6806"]
```

Comment:
would it be helpful to add a comment above the chain IDs with the names of the chains as well.